examples/lock-app/esp32/main/locking_service.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+import 'pw_protobuf_protos/common.proto';
+
+package chip.rpc;
+
+message LockingState {
+  bool on = 1;
```

Comment:
Is ' locked' a more clear name then 'on'?

Reply:
Done